lidar_accelator/scripts/bench_go2_lidar_decode_and_process_sample.py

```
@@ -0,0 +1,132 @@
+    for _ in range(int(args.warmup)):
+        _ = python_end_to_end()
+        _ = cpp_end_to_end()
+
+    py_times, py_out = _bench(python_end_to_end, int(args.iters))
+    cpp_times, cpp_out = _bench(cpp_end_to_end, int(args.iters))
+
+    if py_out.shape != cpp_out.shape or not np.allclose(
+        np.sort(py_out, axis=0), np.sort(cpp_out, axis=0)
+    ):
+        # Sorting per-column is not perfect, but catches obvious mismatches.
+        raise SystemExit("Output mismatch (python vs cpp)")
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)